stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
```

Comment:
Seems fine to just use `bool` as the type for the `shallow` argument to `cmp`. From looking at the code it does technically support any object that implements `__bool__` (which is all objects), but many existing stubs use `bool` for arguments that don't strictly need to be bools.